src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -271,6 +271,45 @@ namespace Platform
         }
         return MeasurementFormat::Metric;
     }
+
+    std::string GetSteamPath()
+    {
+        const char* steamRoot = getenv("STEAMROOT");
+        if (steamRoot != nullptr)
+        {
+            return Path::Combine(steamRoot, "ubuntu12_32/steamapps/content");
```

Comment:
I wonder if everything is alright with all those hardcoded `ubuntu12_32` 🤔

Reply:
AFAIK, Steam uses these on every distro.